### ISSUE TITLE
HomeBrew php5.6 install can not reload apache

### ISSUE THREAD

**Issue by topdown** [Homebrew/homebrew-php#1684]
PHP works fine from command line but can't get it to load properly from Mac's apache.
I'm still stuck on Mac OSX 10.7.5 and can't update higher at this time.
With that said, my errors when reloading apache are this
```httpd: Syntax error on line 113 of /private/etc/apache2/httpd.conf: Cannot load /usr/local/Cellar/php56/5.6.7/libexec/apache2/libphp5.so into server: dlopen(/usr/local/Cellar/php56/5.6.7/libexec/apache2/libphp5.so, 10): Library not loaded: /usr/local/lib/libltdl.7.dylib\n  Referenced from: /usr/local/opt/unixodbc/lib/libodbc.2.dylib\n  Reason: Incompatible library version: libodbc.2.dylib requires version 11.0.0 or later, but libltdl.7.dylib provides version 10.0.0```
I tried removing php56 and reinstalling, but same issue.

**Comment by alanthing** [comment]
We've recently pushed PHP 5.6.8. Please completely uninstall PHP 5.6.7, remove the PHP references to module locations in your httpd.conf file, install `brew install php56` *without additional option flags* (it will build both PHP-FPM and mod_php, but you can ignore the PHP-FPM plist and binary, which is very small), and then add the module path shown at the end of installation into your Apache httpd.conf file. 

Even if the build is successful, you may want to post the output of `brew gist-logs php56` here and we can see if we can gain any additional insight.

**Comment by topdown** [comment]
It still installed PHP 5.6.7 with the same errors
Is there a 1 line command to make sure all brew PHP items are removed?

**Comment by alanthing** [comment]
`brew list | grep -E '^php56' | xargs brew uninstall --force` will remove every php56 item.

Please run `brew update` and you'll have the formula for 5.6.8.

**Comment by topdown** [comment]
Somehow I missed brew update/upgrade and had 48 packages outdated.
But, still did not work.
Went back and removed all brew php56 and reinstalled.
php -v was complaining about imagick5..6 missing so I installed it.
Still same error as above.
```brew gist-logs php56```
Does not seem to do anything.

**Comment by topdown** [comment]
Ok, so I installed https://github.com/philcook/brew-php-switcher
Then installed php54, and php55
Still had errors trying to start apache.
BUT: switched to php53 and had errors, then switched back to 5.4, 5.5, and 5.6 with no issues.
I think the linking may have fixed something using the switcher
php5.3 still has an issue

```dyld: Library not loaded: /usr/local/opt/libpng/lib/libpng15.15.dylib
  Referenced from: /usr/local/bin/php
  Reason: image not found
Trace/BPT trap: 5```

So I created a symlink to the one in Cellar which fixed that.

I then ended up with an error
```dyld: Library not loaded: /usr/local/lib/libodbc.2.dylib
  Referenced from: /usr/local/bin/php
  Reason: image not found
Trace/BPT trap: 5```

Which I couldn't find, so I did a 
```brew reinstall php53```
Then re-ran the version switch tool.
All versions are now working without error 5.3, 5.4, 5.5, 5.6
